enhancements/network/user-defined-network-for-virt-workloads.md

```diff
@@ -0,0 +1,563 @@
+---
+title: primary-user-defined-networks-for-virtualization-workloads
+authors:
+  - @maiqueb
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - @trozet
+  - @jcaamano
+  - @qinqon
+  - @edwardh
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
+api-approvers:
+  - @dougbtv
+creation-date: 2024-08-30
+last-updated: 2024-08-30
+tracking-link:
+  - https://issues.redhat.com/browse/CNV-46779
+see-also:
+  - "/enhancements/network/user-defined-network-segmentation.md"
+  - "/enhancements/network/persistent-ips-sdn-secondary-networks.md"
+---
+
+# Primary User Defined Networks for Virtualization workloads
+
+## Summary
+
+Live-migration is the bread and butter of any virtualization solution.
+OpenShift Virtualization does not currently support live-migration over the
+OpenShift default cluster network, for a variety of reasons. The scope of this
```

Comment:
This is not true. We do support live migration over the pod network with use of masquerade. I would say we "allow live migration over the cluster network, but it requires a compromise in form of NAT which is making the solution unacceptable for many use-cases".

You may argue that TCP disconnect is not making it "live", but except that one issue we tick all the boxes of a live migration, and it is called live-migration by developers and users alike. So to avoid any confusion, I suggest changing this sentence.

Reply:
I argue that it's not only TCP disconnect, but TCP disconnect without **any** chance of **ever** reconnecting unless the clients go and learn the new IP to contact the VM.